terraform/zones/zone.library.ucsb.edu.tf

```
@@ -1053,12 +1053,15 @@ zone_id = local.library-zone_id
   records = ["haproxy.library.ucsb.edu."]
 }
 
-resource "aws_route53_record" "adp-library-ucsb-edu-CNAME" {
+resource "aws_route53_record" "adp-library-ucsb-edu-A" {
 zone_id = local.library-zone_id
   name    = "adp.library.ucsb.edu."
-  type    = "CNAME"
-  ttl     = "10800"
-  records = ["haproxy.library.ucsb.edu."]
+  type    = "A"
+  alias {
+    name                   = data.aws_lb.dld-eks-ingress-nginx-v3.dns_name
+    zone_id                = data.aws_lb.dld-eks-ingress-nginx-v3.zone_id
+    evaluate_target_health = true
+  }
```

```suggestion
  type       = "cname"
  ttl          = "360"
  records  = [ "adp.eks.dld.library.ucsb.edu." ]
```
this will circumvent having to change the hostname on the ingress, so the dns change will be all that's needed for cutting over

---

Let me see if I understand...

so your suggestion would result in changing a CNAME record instead of an A record?

And how will that affect ingress and TLS?